galaxy/accounts/models.py

```diff
@@ -86,15 +86,15 @@ class CustomUser(auth_models.AbstractBaseUser,
         default=False,
         help_text='Notify me when an import succeeds.')
     notify_content_release = models.BooleanField(
-        default=False,
+        default=True,
```

Comment:
Not related to this patch, but usually it's good idea to keep Django user model minimal and use related models for extending it.